src/Config.php

```diff
@@ -285,27 +285,23 @@ private static function init()
 
         switch (PHP_OS_FAMILY) {
             case "Windows":
-                $library_ext = ".dll";
+                // win needs the ABI number before the .dll
+                $library_ext = "42.dll";
                 break;
 
             case "OSX":
                 $library_ext = ".dylib";
```

Comment:
```suggestion
                $library_ext = ".42.dylib";
```
(untested)

Reply:
You're right. I've now copy-pasted (more or less) what pyvips and rubyvips do.